src/client/type_urls.rs

```diff
@@ -23,3 +23,15 @@ pub const MSG_UNDELEGATE_TYPE_URL: &str = "/cosmos.staking.v1beta1.MsgUndelegate
 
 // ibc msgs
 pub const MSG_TRANSFER_TYPE_URL: &str = "/ibc.applications.transfer.v1.MsgTransfer";
+
+// governance proposals
```

Comment:
Since I added some governance proposal type urls here it made sense to rename the file from msgs.rs -> type_urls.rs